test/unit/fix-request-body.spec.ts

```diff
@@ -52,6 +52,40 @@ describe('fixRequestBody', () => {
     expect(proxyRequest.write).toHaveBeenCalledWith(expectedBody);
   });
 
+  it('should write when body is not empty and Content-Type is multipart/form-data', () => {
+    const proxyRequest = fakeProxyRequest();
+    proxyRequest.setHeader('content-type', 'multipart/form-data');
+
+    jest.spyOn(proxyRequest, 'setHeader');
+    jest.spyOn(proxyRequest, 'write');
+
+    const body = new FormData();
+    body.append('someField', 'some value');
+
+    fixRequestBody(proxyRequest, { body } as Request<express.Request>);
+
+    const expectedBody = handlerFormDataBodyData('multipart/form-data', body);
```

Comment:
Test will always pass if you use the private `handlerFormDataBodyData` function to generate the expectedBody for assertion.

Please use hardcoded values or jest's inline snapshot to assert the `write` call signature.